Satori.AppServices.Tests/DailyStandUps/ActivitySummaryDailyStandUpTests.cs

```diff
@@ -68,13 +96,16 @@ public async Task NoWorkItem()
     {
         //Arrange
         var entry = BuildTimeEntry();
-        entry.Description = null;
+        entry.Description = "meetings";
```

Comment:
This test doesn't need to change to include a time entry description.  Leaving this null is still a valid test.

Reply:
The `null` Description is covered by a test below: `NoTimeEntryComments`